gnss_ndt_selector/config/parameters.yaml

```diff
@@ -7,8 +7,18 @@ score_upper_limit: 60.0
 # num of message thredhold to determin if ndt node is reliable or not
 unreliable_message_upper_limit: 3
 
+# Integer: Maximum allowed time for failed gnss before using ndt_pose
+# Units: ms
+gnss_pose_timeout: 2000
+
+# Integer: Maximum allowed time for failed ndt before using gnss_pose
+# Units: ms
+ndt_pose_timeout: 2000
```

Comment:
Lets make this 1500. 
2s is a long time and I think even in poor localization areas we were able to keep the computation time below 1s